torch_xla/core/dynamo_bridge.py

```diff
@@ -28,6 +28,25 @@
 ptxla_debug = int(os.environ.get('PT_XLA_DEBUG', '0')) == 1
 
 
+class AliasWithBufferDonorContext(object):
+
+  def __init__(self, should_alias: bool):
+    self.should_alias = should_alias
+
+  def __enter__(self):
```

Comment:
I strongly recommend using `contextlib.contextmanager` for trivial contexts like this: https://docs.python.org/3/library/contextlib.html#contextlib.contextmanager